opennms-doc/guide-admin/src/asciidoc/text/service-assurance/monitors/MailTransportMonitor.adoc

```diff
@@ -0,0 +1,182 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../../images
+
+==== MailTransportMonitor
+
+MailTransportMonitor is used to run a synthetic test of a complete email transaction, including sending a mail and determining that it has been delivered.  It can also use both sendmail-test and readmail-test 
```

Comment:
Text is great. I've been trying to standardize so that there is a separate line for each sentence.